salt/transport/zeromq.py

```diff
@@ -778,7 +782,7 @@ def handle_message(self, stream, payload):
         else:
```

Comment:
Maybe the helper function should also wrap `ret` in `self._encript_private` but I don't know when this should be the case. Please let me know if you have any idea.

Reply:
Your code looks okay here.